src/cgutils.cpp

```diff
@@ -463,7 +463,11 @@ static void jl_dump_shadow(char *fname, int jit_model, const char *sysimg_data,
         CodeGenOpt::Aggressive // -O3
         ));
 
+#ifdef LLVM38
+    legacy::PassManager PM;
```

Comment:
we have `using namespace llvm::legacy;` at the top of codegen.cpp so that this clutter shouldn't be necessary. did that break in LLVM38 for some reason?